SepulcherOfTheFirstOnes/Locales/frFR.lua

```diff
@@ -82,80 +82,80 @@ end
 
 L = BigWigs:NewBossLocale("Halondrus the Reclaimer", "frFR")
 if L then
-	--L.seismic_tremors = "Motes + Tremors"
-	--L.earthbreaker_missiles = "Missiles"
-	--L.crushing_prism = "Prisms"
-	--L.prism = "Prism"
+	L.seismic_tremors = "Secousses Sismiques"
```

Comment:
This translation is supposed to represent 2 different abilities, but you removed the `+` why?

Reply:
An oversight, with hindsight I was able to find another translation,  Adding it in the next pull request.